### PR TITLE
Chore: LANDGRIF-270 Refactor Data-Model

### PR DESCRIPTION
**Pull request by alexeh**
Update relations
Fix data-import
Add risk-map by resolution
Simplify h3 generation flow and delete Risk-map module

### CODE REVIEW

File: api/src/modules/materials/material.entity.ts [hunk 1]
```diff
@@ -112,25 +112,28 @@ export class Material extends TimestampedBaseEntity {
...
-  /**
-   * @note: Although there should not be any material with no h3GridId, this needs to be nullable
-   * since materials needs to exist before the h3 script takes care of adding these IDs to
-   * each material
-   */
+  @JoinColumn()
+  harvest: H3Data;
   @Column({ nullable: true })
-  h3GridId!: string;
+  harvestId: string;
```
Comment (tiagojsag) [inline api/src/modules/materials/material.entity.ts:138]: meta question: why "harvest"? what if the material is a mineral or something else?

File: api/src/modules/materials/materials.controller.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ import { CreateMaterialDto } from 'modules/materials/dto/create.material.dto';
 import { UpdateMaterialDto } from 'modules/materials/dto/update.material.dto';
 import { MaterialRepository } from 'modules/materials/material.repository';
 import { ApiOkTreeResponse } from 'decorators/api-tree-response.decorator';
-import { ParseOptionalIntPipe } from 'pipes/parse-optional-int.pipe';
+import { ParseOptionalIntPipe } from 'pipes/parse-optional-int.pipe';import { Unit } from 'modules/units/unit.entity';
```
Comment (tiagojsag) [inline api/src/modules/materials/materials.controller.ts:35]: wtf

File: api/test/risk-map/risk-map.spec.ts [hunk 3]
```diff
@@ -107,62 +110,65 @@ describe('Risk Map Test Suite (e2e)', () => {
...
-      h3GridId: indicatorH3GridId,
     });
+    const indicatorH3Data = await createFakeH3Data(
+      INDICATOR_FAKE_H3_TABLE,
+      INDICATOR_FAKE_H3_COLUMN,
+      h3IndicatorFixtures,
+      indicator.id,
+    );
 
     const response = await request(app.getHttpServer()).get(
```
Comment (tiagojsag) [inline api/test/risk-map/risk-map.spec.ts:170]: use .query() instead if possible, makes code easier to maintain

**Review by tiagojsag — CHANGES_REQUESTED** [review]
a few minor things here and there, overall LGTM once those are taken care of

File: api/src/modules/materials/material.entity.ts (hunk 1 above)
Comment (alexeh) [reply api/src/modules/materials/material.entity.ts:138]: meta answer: naming does not reference the 'kind' of the material, but the type of the dataset, which we use then to determine which kind of calculus we need to do:
-Material Map -> producer
-Risk-Map -> harvest

Also, it is required by data to link existing materials to magically imported h3 without the API being involved

File: api/src/modules/materials/materials.controller.ts (hunk 2 above)
Comment (alexeh) [reply api/src/modules/materials/materials.controller.ts:35]: **alex grabbed his backpack and never returned. He was seen living naked in the mountains**

**Pull request merged**